lib/gds_design_system_breadcrumb_builder.rb

```diff
@@ -6,7 +6,7 @@
 #
 class GdsDesignSystemBreadcrumbBuilder < BreadcrumbsOnRails::Breadcrumbs::Builder
   def render
-    @context.tag.div(class: 'govuk-breadcrumbs') do
+    @context.tag.div(class: 'govuk-breadcrumbs', role: 'navigation', 'aria-label': 'Navigate Case') do
```

Comment:
Nit: Could the arial-label value be internationlized using `I18n.t()`?

Reply:
Will add a tech debt ticket for this as there are many other similar examples in different parts of the code where a string is used instead of locale text